src/payment/strategies/klarna/klarna-credit.ts

```diff
@@ -27,3 +32,19 @@ export interface KlarnaAuthorizationResponse {
         invalid_fields: string[];
     };
 }
+
+export type KlarnaUpdateSessionParams = Partial<{
```

Comment:
I don't think defining a type as a partial is recommended. What you had was probably better. 

What you could do instead is make `_getUpdateSessionData` return `KlarnaUpdateSessionParams | undefined` and update
```javascript
load(params: KlarnaLoadParams, data: KlarnaUpdateSessionParams, callback: (res: KlarnaLoadResponse) => void): void;
```
to 
```
load(loadParams: KlarnaLoadParams, updateParams: KlarnaUpdateSessionParams | undefined, callback: (res: KlarnaLoadResponse) => void): void;
```

Reply:
My first approach was `load(loadParams: KlarnaLoadParams, updateParams: KlarnaUpdateSessionParams | {}, callback: (res: KlarnaLoadResponse) => void): void;` but I changed it as suggested by Ignacio

Reply:
yeah, what you could have done is define the argument as partial, not the type itself, e.g:

```javascript
load(loadParams: KlarnaLoadParams, updateParams: Partial<KlarnaUpdateSessionParams>, callback: (res: KlarnaLoadResponse) => void): void
```

Usually is not advised to define the type itself as a partial it might be harder to make it reusable in the future. The stricter your types are the better.